langs/it.json

```diff
@@ -38,7 +42,4 @@
     }
   },

```

Comment:
Invalid JSON because of this trailing comma here as well.

Reply:
Sorry, tried to do this with the web github editor and got confused